assets/js/bgseo.js

```diff
@@ -752,9 +752,12 @@ BOLDGRID.SEO = BOLDGRID.SEO || {};
 				contentEditor = self.getTinymce();
 
 			if ( ! contentEditor || contentEditor.isHidden() ) {
-				text =  api.Editor.element.val();
+				text = api.Editor.element.val();
 			} else {
 				text = contentEditor.getContent( { format: 'raw' } );
+				
+				// Sometimes, on initial page load, the 'getContent' method doesn't return anything.
```

Comment:
> // Sometimes, on initial page load, the 'getContent' method doesn't return anything.

Do we know why this is? Are we adding a band-aid because we're not quite sure what's happening?

Reply:
QA has tested this.

This is just because of timing.... the value of the `api.Editor.element` is present as soon as the page is loaded, but the `contentEditor.getContent()` method returns the contents AFTER they've been loaded into the tinyMCE application.

The preference is to use the text  returned by the `getContent()` method, because it more accurately reflects changes as the user is making them AFTER the initial page load. However the usage of the `api.Editor.element.val()` is totally acceptable as a fallback for when the `getContent()` method is not populated yet.